File: reviews_service/src/main/java/upm/tfg/b190222/reviews_service/entity/Review.java
package upm.tfg.b190222.reviews_service.entity;

import java.time.LocalDate;


import jakarta.persistence.Column;
import jakarta.persistence.Table;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Getter
@Setter
@NoArgsConstructor
@Table(name="Review")
public class Review {
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name="idReview")
    private int idReview;

    @Column(name="username")
    private String username;

    @Column(name="videojuego")
    private String videojuego;

    @Column(name="titulo")
    private String titulo;

    @Column(name="nota")
    private int nota;

    @Column(name="comentario")
    private String comentario;

    @Column(name="fecha")
    private LocalDate fecha;
}

File: FrontEnd/src/app/reviews/review.ts
export class Review {
    idReview:number|null;
    username:string|null;
    videojuego:string|null;
    titulo:string|null;
    comentario:string|null;
    nota:number|null;
    fechaRegistro:Date|null;

    constructor(){
        this.idReview = null;
        this.username = null;
        this.videojuego = null;
        this.titulo = null;
        this.comentario = null;
        this.nota = null;
        this.fechaRegistro = null;
    }
}
